lib/rules/indent.js

```diff
@@ -1229,7 +1229,7 @@ module.exports = {
             },
 
             Property(node) {
-                if (!node.computed && !node.shorthand && !node.method && node.kind === "init") {
+                if (!node.shorthand && !node.method && node.kind === "init") {
```

Comment:
To be honest, I'm not sure why I put this condition in originally (it's been there since the [initial indent rewrite commit](https://github.com/eslint/eslint/blob/12832735450909dbebd1fad1e8ab7c47be67b7f4/lib/rules/indent.js#L774)). I don't think I would have added the condition unless its absence was causing tests to fail, but since all the tests are now passing after removing it, I'm assuming that issue was resolved.